src/trace-span.js

```diff
@@ -16,12 +16,18 @@
 
 'use strict';
 
+var constants = require('./constants.js');
+
 /**
  * Creates a trace span object.
  * @constructor
  */
 function TraceSpan(name, spanId, parentSpanId) {
-  this.name = name;
+  if (name.length > constants.TRACE_SERVICE_SPAN_NAME_LIMIT) {
```

Comment:
`name.length` is the number of unicode code units in the string `name` not bytes.

Can you add a testcase where the code units are < 128, but the byte length is greater?